solidity/storage.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+OUTPUT_PATH=${1:-storage}
+EXCLUDE="test|mock|interfaces|libs|upgrade|README|Abstract|Static"
```

Comment:
we don't want StaticMultisigIsm, aggregationIsm, etc - they have a moduleType in storage?

Plus, I named it StaticProtocolFee which is needs the storage layout.

Reply:
`moduleType` not in storage
`Static` is the name we have used for constants in bytecode
will rename `StaticProtocolFee` so we capture the storage there

Reply:
done in 1141ce513d96e4847d5c7f4b65040da0c59d40e6